.travis.yml

```diff
@@ -41,7 +24,11 @@ matrix:
         - cmake -H. -Bout/darwin/x64/release -DESCARGOT_HOST=darwin -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=release -DESCARGOT_OUTPUT=bin -DVENDORTEST=1 -GNinja
         - ninja -Cout/darwin/x64/release
         - cp ./out/darwin/x64/release/escargot ./escargot
-        - tools/run-tests.py --arch=x86_64 v8
+        - travis_wait tools/run-tests.py --arch=x86_64 jetstream-only-simple
+        - tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js test262 internal jsc-stress regression-tests v8 
+        # FIXME: chakracore fails on darwin
+        # ChakraCore's test runner uses `readlink -f` to determine the test root directory.
+        # However, `readlink` has no `-f` option on macOS.
```

Comment:
If octane is not tested on darwin, I think it would be worth mentioning why, like it is documented for chakracore and spidermonkey.

Reply:
Both octane and jetstream-only-simple are excluded from darwin target at now. I mentioned this issue in travis file. Thank you for the review.